### DISCUSSION TITLE
joomvideo and jooma cck

### DISCUSSION THREAD

**Discussion by spinelbusiness** [JoomCoder-com/JoomCCK#122] [Ideas]
I've just heard that joomcck is out, it's incredible, you should communicate about it. 
2 questions: 1) is forum template already available on cobalt and its plugins still there?
2) is joom video integrated with joom cck?
thanks

**Comment by andiamine** [comment]
`1) is forum template already available on cobalt and its plugins still there?
`
not yet, but we are going to make back all missed packs not only forums pack.

`2) is joomvideos integrated with joomcck?`

We can do as custom work, give us more details about how you want this integration at joomcoder.com

**Comment by JEROMELECORSE** [comment]
joomvideos in joomlacck would be amazing, cobalt lacked in video module
